### ISSUE TITLE
Undo action for point placement when drawing Spline

### ISSUE THREAD

**Issue by MiltonBalaOfficial** [xournalpp/xournalpp#5943]
### Is Your Feature Request Related to a Problem?

While drawing a spline if you want to undo the last point addition it is not possible.

### Feature Description

add undo action for the point addition in spline

**Comment by rolandlo** [comment]
There is no undo action while editing the spline (because the document is still unchanged), but you can press the backspace key to undo the last point addition.

**Comment by complanar** [comment]
Maybe related: Is there a possibility to edit control points of a spline or straight line afterwards?

**Comment by rolandlo** [comment]
No, that's currently not possible. Once splines are finished and inserted into the document, they become mere strokes and loose the information about control points. The file format doesn't support splines yet, although in (maybe distant) future it might.

**Comment by complanar** [comment]
Just digged into the file format a little bit and found out that straight lines are just very simple strokes with two points. Thus creating a tool which determines the number of points a stroke consists of and then allows dragging them around should be possible. Shall I open a new issue/feature request for that?

**Comment by Prateek-coder19** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
